lib/util/glob.js

```diff
@@ -24,40 +25,39 @@ const IGNORE = typeof Symbol === "function" ? Symbol("ignore") : "_shouldIgnore"
  * @param {function()} shouldIgnore Method to check whether a directory should be ignored.
  * @constructor
  */
-function GlobSync(pattern, options, shouldIgnore) {
-
-    /**
-     * We don't put this thing to argument `options` to avoid
-     * further problems, such as `options` validation.
-     *
-     * Use `Symbol` as much as possible to avoid confliction.
-     */
-    this[IGNORE] = shouldIgnore;
-
-    Sync.call(this, pattern, options);
-}
-
-util.inherits(GlobSync, Sync);
+class GlobSync {
```

Comment:
Since `GlobSync` inherits from `Sync`, I think it would be better to use `class GlobSync extends Sync` here.